stock_partner_version/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion - Benoît Guillot
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Partner Version',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
```

Comment:
Please change at least this using https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py